packages/betterer/src/printer.ts

```diff
@@ -16,8 +17,10 @@ export function print(results: BettererResults): string {
           ? (value as Printable).print()
           : JSON.stringify(value);
     }
-    printedValue = (printedValue as string).replace(/`/g, '\\`');
-    return `\nexports[\`${resultName}\`] = {\n  timestamp: ${timestamp},\n  value: \`${printedValue}\`\n};`;
+    return `\nexports[\`${resultName}\`] = {\n  timestamp: ${timestamp},\n  value: \`${escape(
+      printedValue as string,
```

Comment:
2. When we're outputting `value` from the printer, we also want to ensure it's safe for `JSON.parse`. In this case we want to make sure any characters that might break the value string are escaped. While we _did_ escape `message` from `file-info.ts`, other things can (and do) break value. 

👉 [Continue]()